packages/fluent-theme/.vscode/settings.json

```diff
@@ -0,0 +1,1984 @@
+{
```

Comment:
What's up with this giant file? Is it needed?

Reply:
It's created from #6719's create package script.. @kkjeer, do we need all of this content for new packages?

Reply:
I added it since our other packages have .vscode/settings.json, but it's probably not strictly necessary.

Reply:
@kkjeer shouldn't this file be just at the root of the repo?

Reply:
Currently we have multiple settings.json files in several of our packages (charting, experiments, foundations, office-ui-fabric-react, and fabric-website). Going forward, it doesn't seem like we should need one in every new package, so I removed the template from the create-package PR #6719.

Reply:
In fact I wonder if we should go through and remove all of these files.. I'm curious why we would need to affect user settings at all in something I'm guessing is autogenerated (unless they somehow affect linting, TS, etc.)

Reply:
I don't think they affect user settings. I see that it's mostly to setup some workspace rules for a specific repo. User settings can be changed only locally in I am not wrong.
So the gist is that only the root folder of the repo needs to have it (which it has) and everything else is just duplicating and could possibly create inconsistencies in between packages present in the repo if some package decides to override something. Not sure if this is how it works though. Also, I wonder if any of this settings could interfere with the Prettier...

Reply:
This is for people who open the individual packages directly. @ecraig12345 does that.

Reply:
Does anyone know if having one at the root folder and then for each package another one... will they interfere in any way? Is VSCode smart enough to ignore those ones if the workspace root folder has one already? If that's the case then I think we actually should add an exact match of the one at the root folder of the repo to each package for cases @cliffkoh mentioned. They should not differ and be identical for consistency.